src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -2,18 +2,29 @@
 
 public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-	int totalRejected = 0;
+	private static final int ZERO = 0;
 
-	public int reportRejected() {
-		return totalRejected;
+    private int totalRejected = ZERO;

```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.